### PR TITLE
Improve install doc

### LINKED ISSUE #1329: rosa utility to install configuration metadata for site
We should provide a `rosa` utility to install configuration metadata for site.

### PR DESCRIPTION
**Pull request by matthewrmshin**
How to test that the built-in Rosie web service server is up and
running.

How to deploy site configuration metadata. Close #1329.

### CODE REVIEW

File: doc/rose-install.html [hunk 1]
```diff
@@ -508,20 +534,66 @@ <h2 id="rose-bush">Configuring Rose Bush</h2>
...
-    <h2 id="meta">Storing Configuration Metadata</h2>
+    <h2 id="meta">Deploying Configuration Metadata</h2>
+
+    <p>You may want to deploy <a href=
+    "rose-configuration-metadata.html">configuration metadata</a> for projects
+    using Rose in a globally readable location at your site, so that they can be
+    easily accessed by users when using Rose utilities such as <code>rose
+    config-edit</code> or <code>rose macro</code>.</p>
+
+    <p>If the source tree of a project is versioned controlled under a trusted
```
Comment (arjclark) [inline doc/rose-install.html]: version~~ed~~ controlled

**Comment by arjclark** [comment]
Just the one typo. Otherwise looks good to me.

**Comment by matthewrmshin** [comment]
Typo fixed.

**Comment by arjclark** [comment]
Ta. Looks good to go.

**Pull request merged**